rn/Teacher/ios/Teacher/Submissions/QuizSubmissionList/Model/Entities/QuizSubmissionListItem.swift

```diff
@@ -0,0 +1,71 @@
+//
+// This file is part of Canvas.
+// Copyright (C) 2023-present  Instructure, Inc.
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as
+// published by the Free Software Foundation, either version 3 of the
+// License, or (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+//
+
+import Core
+
+public struct QuizSubmissionListItem: Equatable {
+    public let id: String
+    public let displayName: String
+    public let name: String
+    public let status: QuizSubmissionWorkflowState
+    public let score: String?
+    public let avatarURL: URL?
+
+    public static func generateArray(users: [QuizSubmissionUser], submissions: [QuizSubmission]) -> [QuizSubmissionListItem] {
```

Comment:
The `generateArray` name sound strange to me. I would simply use `make` instead. The `array` suffix in the method name feels redundant since we know from the return type that it's returning an array.